vowpal_wabbit/ubuntu1604-build.Dockerfile

```diff
@@ -45,6 +45,17 @@ RUN version=3.13 && build=5 \
  && for filename in /opt/cmake/bin/*; do echo Registering $filename; ln -fs $filename /usr/local/bin/`basename $filename`; done \
  && rm cmake-$version.$build-Linux-x86_64.sh
 
```

Comment:
nit: please add a comment just for consistency sake with the file, I know they are not very descriptive but to keep the format